go.mod

```diff
@@ -69,7 +69,6 @@ require (
 	github.com/jbenet/goprocess v0.1.4
 	github.com/libp2p/go-doh-resolver v0.4.0
 	github.com/libp2p/go-libp2p v0.19.1
-	github.com/libp2p/go-libp2p-connmgr v0.3.2-0.20220115145817-a7820a5879c7 // indirect
```

Comment:
This repo is now archived and part of go-libp2p. The relevant commit here is in go-libp2p v0.19.1 https://github.com/libp2p/go-libp2p/commit/e32c45849c28388a74392d509715d138c7691607